simple_history/tests/tests/test_models.py

```diff
@@ -1478,3 +1483,46 @@ def test_revert(self):
 
         self.one = Street.objects.get(pk=id)
         self.assertEqual(self.one.history.count(), 4)
+
+
+@override_settings(**database_router_override_settings_history_in_diff_db)
+class SaveHistoryInSeparateDatabaseTestCase(TestCase):
+    multi_db = True
+
+    def setUp(self):
+        self.model = ModelWithHistoryInDifferentDb.objects.create(name="test")
+
+    def test_history_model_saved_in_separate_db(self):
+        self.assertEqual(0, self.model.history.using("default").count())
+        self.assertEqual(1, self.model.history.count())
+        self.assertEqual(1, self.model.history.using("other").count())
+        self.assertEqual(
+            1, ModelWithHistoryInDifferentDb.objects.using("default").count()
+        )
+        self.assertEqual(1, ModelWithHistoryInDifferentDb.objects.count())
+        self.assertEqual(
+            0, ModelWithHistoryInDifferentDb.objects.using("other").count()
```

Comment:
Similarly - the base model table shouldn't exist in other?

Reply:
@rossmechanic explains this in more detail above, but the TLDR is that the migrations for tests are hard to tailor on a per-test basis. These tests ensure that history can be written to and read from separate databases than their associated operational models.